src/app/organizer/Columns.tsx

```diff
@@ -588,9 +706,11 @@ export function getColumns(
         header: t('Organizer.Columns.Level'),
         value: (item) => item.craftedInfo?.level,
         defaultSort: SortDirection.DESC,
+        csvVal: (value) => ['Crafted Level', value ?? 0],
```

Comment:
In general I want columns to return `undefined` for their value if the column doesn't apply. But then annoyingly the CSV was inconsistent about whether a column was undefined or zeroed.